bridge_adaptivity/module/tests/test_views.py

```diff
@@ -377,3 +377,22 @@ def test_update_not_mine_course(self):
         self.assertEqual(course.name, self.course.name)
         self.assertEqual(course.description, self.course.description)
         self.assertEqual(course.owner, self.course.owner)
+
+
+class TestManualSync(BridgeTestCase):
+
+    @patch('module.tasks.sync_collection_engines.delay')
+    @patch('module.tasks.sync_collection_engines.apply_async')
+    @patch('module.views.get_available_courses')
+    def test_immediate_synchronization(
+        self, mock_get_available_courses, mock_apply_async, mock_delay
+    ):
+        col_id = self.collection1.id
+        expected_url = reverse('module:collection-detail', kwargs={'pk': col_id}) + '?back_url=None'
+        url = reverse('module:collection-sync', kwargs={'pk': col_id})
```

Comment:
what about negative test case? 

what we will have if someone pass not correct pk?
I would add it just to be sure that it does not fall in 500.

Reply:
done